requirements.txt

```diff
@@ -0,0 +1,29 @@
+pyside
```

Comment:
Xi-cam uses setup.py for the requirements; see `installrequires = [...]`. Lets merge these into there. It looks like you've added the following dependencies: 

> sklearn, six, matplotlib, silx, pypaws>0.8.4, pypif, pytz, tzlocal, saxskit

Are these required for the batch plugin and paws?

Reply:
Also, should it be `pypaws>=0.8.4`? The latest I see in pip is 0.8.4.

Reply:
Yes- your'e right about the version.

As for the long comment in __init__, I actually didn't mean for that comment to be a docstring. 

I messed up those dependencies a bit (copy pasted them from elsewhere)- we actually don't need most of those.

I'll push a commit shortly for all of the above.